ulauncher/modes/extensions/extension_remote.py

```diff
@@ -34,6 +34,7 @@ class ExtensionNetworkError(Exception):
 class UrlParseResult(BaseDataClass):
     ext_id: str
     remote_url: str
+    browser_url: str | None = None
```

Comment:
It's optional and won't be set if we can't derive the browser url from the input url (unless it's on GitHub, GitLab or Codeberg, the input url must be a remote that can be cloned with `git`, and then it won't be a web url also)

Reply:
Also, note that this URL can still be a file url (`FILE:///...`), which is why I named it browser_url, not web_url. Both work in a browser/can be browsed, but file urls are not web urls.